.drone.yml

```diff
@@ -34,7 +34,9 @@ pipeline:
 
 matrix:
   TEST:
-    - ;test;dotty-bin-tests/test
-    - ;publishLocal;dotty-bootstrapped/test
-    - partest-only-no-bootstrap --show-diff --verbose
-    - partest-only --show-diff --verbose
+    - ;set testOptions in LocalProject("dotty-compiler") += Tests.Argument(TestFrameworks.JUnit, "--exclude-categories=dotty.tools.dotc.ParallelTesting") ;test ;dotty-bin-tests/test
```

Comment:
True, also if it wasn't clear - they will be removed once we remove partest itself.